voila/tornado_handler.py

```diff
@@ -0,0 +1,20 @@
+#############################################################################
```

Comment:
I think we could move these 2 files into the `tornado` directory, so we can remove the prefix in the file names.

Reply:
:+1: